src/lib/Runner.ts

```diff
@@ -761,7 +774,13 @@ export class Runner {
     });
     return landRequestStatus;
   };
-
+  // filter queue by target branch - used by speculation engine to reorder PRs by target branch
+  filterQueue = async (landRequest: LandRequest) => {
```

Comment:
nit: can be renamed to `filterQueueByTargetBranch` and the input of the function can be `targetBranch`